training/distributed_training/tensorflow/model_parallel/mnist/tensorflow_smmodelparallel_mnist.ipynb

```diff
@@ -8,15 +8,14 @@
     "\n",
```

Comment:
<p>Is the correct branding <code>Distributed</code>  or <code>distributed</code>?  There is a mix within the files in this PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3043/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>